### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by demsytohoku**
I update the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # test-pull-request
 Just a repo to practice and familiarize with pull request
+
+# This is add by demsy tohoku
```
Comment (orvindemsy) [inline README.md:4]: Hmm could you add more comments why you add this line

Comment (demsytohoku) [reply README.md:4]: I add this line because I am you

**Review by demsytohoku — COMMENTED** [review]
What happen if I click this button

**Review by demsytohoku — COMMENTED** [review]
It's weird that I as requester can submit review

File: README.md (hunk 1 above)
Comment (orvindemsy) [inline README.md:4]: How about if I add new comment here

Comment (orvindemsy) [inline README.md:4]: Add another one here but click 'Start a review'

**Review by orvindemsy — COMMENTED** [review]
I will add another comment

**Review by orvindemsy — COMMENTED** [review]
Submit a another review from orvindemsy

**Review by orvindemsy — CHANGES_REQUESTED** [review]
I request to change the line 5 waitt there't no line 5

**Pull request merged**